samples/smoketest/README.md

```diff
@@ -0,0 +1,116 @@
+
+Libraries tested:
+* keyvault-secrets
+* identity
+* storage-blob
+* event-hubs
+* cosmos
+
+## Getting started
+### Setup Azure resources
+For this sample, it is necessary to create/have the following resources in the [Azure Portal](https://portal.azure.com/):
+* **App registration**: Register a new app or use an existing one.
+  * Under _Certificates & secrets_ create a new **client secret** and store the value in a safe place.
+* **Key Vaults**: Create a new Key Vault resource or use an existing one.
+  * Under _Access policies_, add the app registrated in the previous step.
+* **Storage acounts**: Create a container in a new or existing storage account. The container in this sample is named "mycontainer", if you want to use other name you can change the value in `BlobStorage.ts` file:
+`const containerName = "mycontainer";`
+* **Event Hubs**: Create an event hub inside a new or existing Event Hubs Namespace. The container in this sample is named "myeventhub", if you want to use other name you can change the value in `EventHubsTest.ts` file: `let eventHubName = "myeventhub";`
+* **Azure Cosmos DB**: Create a new account or use an existing one.
+
+### Azure credentials
+The following environment variables are needed:
+* From **App Registration**, in the _Overview_ section:
+    * AZURE_TENANT_ID: The directory tentant ID.
+    * AZURE_CLIENT_ID: The application ID.
+    * AZURE_CLIENT_SECRET: The client secret stored previusly when creating the _client secret_.
+
+* From **Key Vault**, in the _Overview_ section:
+  * AZURE_PROJECT_URL: The DNS Name
+
+* From **Event Hubs**, in _Shared access policies_ section:
+  * EVENT_HUBS_CONNECTION_STRING: Connection string from a policy
+
+* From **Storage Account**, in the _Access Keys_ section:
+  * STORAGE_CONNECTION_STRING : A connection strings.
+
+* From **Azure Cosmos DB**, in the _Keys_ section, select the _Read-Write Keys_ tab:
+  * COSMOS_ENDPOINT: URI.
+  * COSMOS_KEY: Primary or secondary key.
+
+```
+//Bash code to create the environment variables
+export AZURE_CLIENT_ID=""
+export AZURE_CLIENT_SECRET=""
+export AZURE_TENANT_ID=""
+export EVENT_HUBS_CONNECTION_STRING=""
+export AZURE_PROJECT_URL=""
+export STORAGE_CONNECTION_STRING=""
+export COSMOS_ENDPOINT=""
+export COSMOS_KEY=""
+```
+
+### Running the console app
+[Python](https://www.python.org/downloads/) version 3.7.4 was used to run this sample.
+
+In the \SmokeTest\ directory, run Program.py
+```
+python .\Program.py
+```
+
+## Key concepts
+
+
+## Examples
+All the classes in this sample has a `Run()` method as entry point, and do not depend on each other. 
+
+It is possible to run them individually:
+```python
+from KeyVaultSecrets import KeyVault
+
+KeyVault().Run()
+```
+
+They can be included in other projects by moving the class in it:
+```python
+from KeyVaultSecrets import KeyVault
+
+...
+
+def  myTests():
```

Comment:
nitpicking: extra space in front of `myTests`

Reply:
I get this from a commit already on master (this PR starts from a merge on master), so won't change it to keep this PR focused on Storage.
See: https://github.com/Azure/azure-sdk-for-python/pull/6412